popSimManu.tex

```diff
@@ -227,43 +227,44 @@ \section*{Introduction}
+within individual species (expanded information shown here for \emph{H.~sapiens} only). 
 Each species is associated with a representation of the physical genome, and one or more genetic maps and demographic models.
-Dotted lines indicate that only a subset of these categories is shown. \textbf{(B)} Example code to specify
-and simulate models using \stdpopsim via the API or \textbf{(C)} through a command line interface.}
+Dotted lines indicate that only a subset of these categories is shown. 
+At right we show example code to specify
+and simulate models using \textbf{(B)} the python API or \textbf{(C)} the command line interface.
+    }
 \label{fig:cartoon}
 \end{center}
 \end{figure}
 
 \section*{Results}
-\paragraph{The \stdpopsim library}
+\paragraph{The \stdpopsim library.}
 The first contribution of the PopSim consortium is \stdpopsim, a
 community-maintained library of empirical genome data and population genetics simulation
 models. Figure \ref{fig:cartoon} shows a graphical
 representation of the structure of \stdpopsim. The package centers
-on a catalog of species (Fig. \ref{fig:cartoon}A), initially consisting of humans,
-\emph{D.~melanogaster}, \emph{A.~thaliana}.
-% RNG: Abbreviate consistently with introduction
+on a catalog of species (Fig.~\ref{fig:cartoon}A), initially consisting of humans,
+\textit{D.~melanogaster}, and \textit{A.~thaliana}.
 A species definition consists of two key elements.  Firstly, the library defines
-some basic information about the species' genome, including information about chromosome
-lengths, average mutation rates and so on. We also provide access to detailed
+some basic information about each species' genome, including information about chromosome
+lengths, average mutation rates, and generation times. We also provide access to detailed
 empirical information such as genetic maps, which model observed
 heterogeneity in recombination rate along chromosomes. As such maps are often large,
 we do not distribute them directly with the software, but make them available
 for download in a standard format. When a simulation using such a map is
 requested by the user, \stdpopsim will transparently download the map
-data into a local cache (\texttt{$\sim$/.local/cache/stdpopsim} by default
-on Unix platforms), where it can be quickly retrieved for subsequent
-simulations.
+data into a local cache,
+% (\texttt{$\sim$/.local/cache/stdpopsim} by default on Unix platforms)
+where it can be quickly retrieved for subsequent simulations.
 In the initial version of \stdpopsim we support
 the HapMapII~\citep{international2007second} and
-deCODE~\citep{kong2010fine} genetic maps in humans;
-the \cite{salome2011recombination} map in \emph{A.~thaliana};
-and the \cite{comeron2012many} map in \emph{D.~melanogaster}.
+deCODE~\citep{kong2010fine} genetic maps for humans;
+the \cite{salome2011recombination} map for \emph{A.~thaliana};
+and the \cite{comeron2012many} map for \emph{D.~melanogaster}.
```

Comment:
maybe add a quick sentence here that says "Adding further maps into the resource is trivial"